src/__init__.py

```diff
@@ -1 +1,2 @@
-from src.client import client_instance
+#from src.client import client_instance
+from src.client import _Client
```

Comment:
What is your reasoning behind having Client not be a singleton anymore?

Reply:
In e2e test, object is created before I can specify to use e2e_config.json, so it automatically loads regular config.json

Reply:
Also the program does not start because there is no client_instance to import in main.py.